extensions/frontend-tiles/src/FrontendTiles.ts

```diff
@@ -235,6 +56,7 @@ export const frontendTilesOptions = {
  * @beta
  */
 export function initializeFrontendTiles(options: FrontendTilesOptions): void {
+  const provider = new MeshExportServiceProvider();
```

Comment:
I'm not convinced that creating an instance of a provider here is the best way to accomplish this, but as of now, I believe it's necessary so that we can call `obtainMeshExportTilesetUrl`. Open to any suggestions.